ocrd_lib.sh

```diff
@@ -257,8 +257,12 @@ kitodo_production_task_action() {
       ;;
```

Comment:
I suggest sticking with our usual error logging. Instead of the echo, do

    logger -p user.error -t $TASK "Unknown task action type"